src/components/Leaderboard.tsx

```diff
@@ -94,21 +38,49 @@ const Leaderboard: React.FC<IProps> = ({ content, limit = 100 }) => (
           emoji = ":3rd_place_medal:"
         }
         return (
-          <Item key={idx} to={hasGitHub ? `${githubUrl}${username}` : "#"}>
-            <ItemNumber>{idx + 1}</ItemNumber>
-            <Avatar src={avatarImg} alt={avatarAlt} />
-            <TextContainer>
-              <ItemTitle>{name}</ItemTitle>
-              <ItemDesc>
+          <Link
```

Comment:
Instead of wrapping this component with a Link, lets use the `LinkOverlay` component that Chakra offers for this specific scenario https://chakra-ui.com/docs/components/link-overlay/usage

I did a similar implementation in https://github.com/ethereum/ethereum-org-website/pull/7721